README.rst

```diff
@@ -49,6 +49,12 @@ Global Options
 -c FILE, --config FILE          Read configuration from the given file [default
                                 value: ``config.yml``]
 
+-E FILE, --env FILE             Load environment variables from the given
+                                ``.env`` file.  By default, environment
+                                variables are loaded from the first file named
+                                "``.env``" found by searching from the current
+                                directory upwards.
```

Comment:
I just setup a `$PWD.env` file with secrets I source before running... 

We need a big fat warning here though -- in `datalad` mode or just with pure git users might commit and leak it. Conscientious of the problem user can configure `.gitignore` or add it to `git-annex` while annotating with metadata to not distribute with default `datalad push` or `publish` commands, but it should be for user to decide...

```suggestion
                                directory upwards.  WARNING: Placing in the same directory
                                as output whenever output is to be committed to git must be done
                                with care: either specify the file to be excluded in ``.gitignore`` or
                                place it under git-annex control and set it up with all necessary
                                provisions to avoid public sharing.
```

tune to your liking

Reply:
Warning added.